packages/xdl/src/ProjectAssets.ts

```diff
@@ -60,20 +61,65 @@ export async function resolveGoogleServicesFile(projectRoot: string, manifest: E
  * @param manifest
  * @returns Asset fields that the user has set like ["icon", "splash.image", ...]
  */
-async function getAssetFieldPathsForManifestAsync(manifest: ExpoAppManifest): Promise<string[]> {
+async function getAssetFieldPathsForManifestAsync(manifest: ExpoConfig): Promise<string[]> {
   // String array like ["icon", "notification.icon", "loading.icon", "loading.backgroundImage", "ios.icon", ...]
   const sdkAssetFieldPaths = await ExpSchema.getAssetSchemasAsync(manifest.sdkVersion);
   return sdkAssetFieldPaths.filter(assetSchema => get(manifest, assetSchema));
 }
 
+async function resolveExpoUpdatesManifestAssets({
+  projectRoot,
+  manifest,
+  assetKeyResolver,
+}: {
+  projectRoot: string;
+  manifest: ExpoConfig;
+  assetKeyResolver: (assetPath: string) => Promise<string>;
+}): Promise<void> {
+  const assetSchemas = await getAssetFieldPathsForManifestAsync(manifest);
+  // Get the URLs
+  const assetInfos = await Promise.all(
+    assetSchemas.map(async manifestField => {
+      const pathOrURL = get(manifest, manifestField);
+      if (pathOrURL.match(/^https?:\/\/(.*)$/)) {
```

Comment:
We don't need support for capture groups:
```suggestion
      if (/^https?:\/\//.test(pathOrURL)) {
```